Mathlib/Data/Fintype/Card.lean

```diff
@@ -347,11 +347,9 @@ theorem Fintype.card_lex (α : Type*) [Fintype α] : Fintype.card (Lex α) = Fin
 @[simp] lemma Fintype.card_additive (α : Type*) [Fintype α] : card (Additive α) = card α :=
   Finset.card_map _
 
--- Note: The extra hypothesis `h` is there so that the rewrite lemma applies,
--- no matter what instance of `Fintype (Set.univ : Set α)` is used.
 @[simp]
```

Comment:
```suggestion
-- Note: The extra hypothesis `h` is there so that the rewrite lemma applies,
-- no matter what instance of `Fintype (Set.univ : Set α)` is used.
@[simp]
```